src/main/java/org/jitsi/utils/collections/JMap.java

```diff
@@ -27,6 +27,7 @@
  * taking individual arguments--are in Java 9 and are provided here
  * for convenience.
  */
+@Deprecated
```

Comment:
Add a comment saying that since we now target Java 11+, this should be avoided in favor of using the native Java methods?

Reply:
I'll just remove it now and update the projects.

Reply:
It would mean that jicoco and ice4j need to be updated to Java 11, but if you're good with that, that's fine.

Reply:
No, I don't want to force them to 11 yet, so I used a different API there.